src/common/campaign/common.ts

```diff
@@ -131,3 +135,63 @@ export const getReferenceTags = (
       throw new ValidationError('Unknown campaign type to estimate reach');
   }
 };
+
+export enum CampaignUpdateEvent {
```

Comment:
Does this come from Skadi or is it API only?
Asking because ideally we should leverage proto then

Reply:
This comes from Skadi.